doc/release/upcoming_changes/29492.deprecation.rst

```diff
@@ -0,0 +1,9 @@
+Setting the ``shape`` attribute is deprecated
+-----------------------------------------------
```

Comment:
```suggestion
---------------------------------------------
```
I am a bit surprised the docs build doesn't dislike links not working (and mayb this)